src/cli/cmds/synth.ts

```diff
@@ -74,5 +130,79 @@ async function fetchValidations(): Promise<ValidationConfig[] | undefined> {
   }
 }
 
+async function createHelmScaffolding(apiVersion: string, chartVersion: string, outdir: string) {
+  const tempHelmStructure = createFolderStructure();
+
+  const substituteValues = {
+    apiVersion: apiVersion,
+    version: chartVersion,
+    app: path.basename(path.resolve()),
+  };
+
+  try {
+    await sscaff(tempHelmStructure, outdir, substituteValues);
+  } catch (error) {
+    throw new Error(`An error occurred during Helm chart creation: ${error}`);
```

Comment:
No need - what context / information are you adding here that isn't available in the original error and stacktrace?

Reply:
I was just adding it to make it clear that this particular operation failed. We can know that from the stack trace too, will update it :)